src/layouts/app/translations-mixin.ts

```diff
@@ -60,12 +64,15 @@ export default (superClass: Constructor<LitElement & HassBaseEl>) =>
 
     private _loadTranslationFragment(panelUrl) {
       if (translationMetadata.fragments.includes(panelUrl)) {
-        this._loadResources(panelUrl);
+        this._loadResources(this.hass, panelUrl);
       }
     }
 
-    private async _loadResources(fragment?) {
-      const result = await getTranslation(fragment);
+    private async _loadResources(hass?, fragment?) {
+      const result = await getTranslation(
+        fragment,
+        hass ? hass.language : getLocalLanguage()
```

Comment:
It still feels a bit weird that we pass in `hass` instead of the language. It's only needed during startup no ?

Reply:
Done